src/App.css

```diff
@@ -1,38 +1,91 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+#root {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
 .App {
-  text-align: center;
+  width: 80%;
+  max-width: 1000px;
+  margin-top: 50px;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.app-logo {
+  height: 200px;
+  opacity: 0.7;
+}
+
+.nav-logo {
+  height: 50px;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: space-between;
+  font-size: 16px;
+  line-height: 50px;
+  color: black;
+}
+h1 {
+  font-size: xx-large;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.navbar-text {
+  padding-right: 20px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+ul.menu {
+  display: flex;
+  list-style-type: none;
+  margin-top: 1em;
+  margin-bottom: 1 em;
+  margin-left: 0;
+  margin-right: 0;
+  padding-left: 40px;
+}
+ul.menu li {
+  padding-left: 20px;
+}
+.ninja-stub {
+  height: 20vh;
   display: flex;
-  flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.add-tracker {
+  width: 581px;
+  background: #FFFFFF;
+  border: 1px solid #D0D0D0;
+  box-sizing: border-box;
+  border-radius: 10px;
+  display: flex;
+}
+.add-tracker-input {
+  width: 481px;
+  height: 40px;
+  margin: 10px;
+  font-size: 20px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.add-tracker-input {border:0;outline:0;}
+.add-tracker-input:focus {outline:none!important;}
+
+.add-tracker-button {
+  width: 20%;
+  font-size: 20px;
+  margin: 5px;
+}
+
+.app-tracker-add {
+  display: flex;
```

Comment:
I would recommend also adding `flex-wrap: wrap` property to this one. That way logo will move to the next line on a smaller screen instead of getting cut off.

Reply:
added